### DISCUSSION TITLE
How can i display the two bars in each month section.

### DISCUSSION THREAD

**Discussion by zeeshanansari-danbertech** [Abhinandan-Kushwaha/react-native-gifted-charts#828] [Q&A]
I want to display completed booking and earning bars in each month. how can I display this chart bar?

<img width="565" alt="Screenshot 2024-09-12 at 10 58 48 AM" src="https://github.com/user-attachments/assets/e3648922-5817-4585-bf3b-435e1ffc48db">

**Comment by Abhinandan-Kushwaha** [comment]
Hi @zeeshanansari-danbertech 👋
Thanks for requesting this feature. It is available from versions `1.4.35` onwards. 🎉

To achieve the above chart, you can have a single data array, with alternate values set to secondary using the property `isSecondary: true` inside the data array objects. Then you can set `spacing: 0` for alternate indices (starting from 0,2,4...). Also, you can set the `frontColor` inside each data array objects.

the right side Y-axis can be rendered using the `secondaryYAxis` prop. For customising the right side Y-axis, see [secondaryYAxis in the props table](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/docs/LineChart/LineChartProps.md#secondaryyaxis)

Here's an example-

```js
onst data: barDataItem[] = [
  {value: 26, spacing: 0},
  {value: 600, frontColor: '#655FD5', isSecondary: true},

  {value: 26, spacing: 0},
  {value: 600, frontColor: '#655FD5', isSecondary: true},

  {value: 26, spacing: 0},
  {value: 600, frontColor: '#655FD5', isSecondary: true},

  {value: 100, spacing: 0},
  {value: 500, frontColor: '#655FD5', isSecondary: true},

  {value: 24, spacing: 0},
  {value: 800, frontColor: '#655FD5', isSecondary: true},

  {value: 23, spacing: 0},
  {value: 300, frontColor: '#655FD5', isSecondary: true},

  {value: 26, spacing: 0},
  {value: 600, frontColor: '#655FD5', isSecondary: true},

  {value: 26, spacing: 0},
  {value: 600, frontColor: '#655FD5', isSecondary: true},

  {value: 26, spacing: 0},
  {value: 600, frontColor: '#655FD5', isSecondary: true},
];

return (
  <BarChart
    data={data}
    noOfSections={7}
    stepValue={20}
    secondaryYAxis={{maxValue: 1400}} // for right side Y-axis you can customize its label colors & other properties 
    frontColor={'#5ABF2A'}
    barWidth={10}
    spacing={8}
  />
);
```
The output of the above code is-

<img width="332" alt="Screenshot 2024-09-13 at 2 27 54 AM" src="https://github.com/user-attachments/assets/0503ca90-4dbc-4a3d-aff2-234fea677094">

Please use the latest version of the library.
